src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__day">
+        <div class="calendar__days"></div>
```

Comment:
Need to fix the classes everywhere
```suggestion
        <div class="calendar__day"></div>
```